### PR TITLE
Added change boards toggle - Issue #14

### LINKED ISSUE #14: Add functionality to change boards
You can create dummy boards for the purpose of testing. Focus is to have functionality that will allow players change the current board if there are other boards available

image

If you have any more questions or need more clarity on this issue, endeavour to join the StarkLudo contributors channel (https://t.me/+hnjQooODZOA2M2Rk) on Telegram and the project leads will be there to help.

### PR DESCRIPTION
**Pull request by melnikga**
Closes #14 
I created a functionality for switching boards using context, as an example I added a "Texture" board where the usual colors are replaced with different textures, the form for switching boards is located in the toolbox window

### CODE REVIEW

File: client/src/assets/images/Yellow white wall photo.jpg
Comment (vibenedict) [inline client/src/assets/images/Yellow white wall photo.jpg:1]: Image is too large and will increase size of codebase :)
Please downscale to less than 250kb or replace with smaller image

**Review by vibenedict — COMMENTED** [review]
Hi @melnikga, great job here but the task is to be able to replace the entire board with another board. The board and the game pieces are not attached together, making it possible to make the board replaceable. 

To better understand the task, comment out `<LudoTiles/>` in [`Ludo.tsx`](https://github.com/sivicstudio/starkludo/blob/dev/client/src/components/Ludo.tsx#L20) and notice how the board vanishes while the game pieces are still in place.

**Comment by melnikga** [comment]
Hey @vibenedict, is the task to replace only this square component?

image

File: client/src/assets/images/Yellow white wall photo.jpg
Comment (vibenedict) [reply client/src/assets/images/Yellow white wall photo.jpg:1]: @melnikga

**Review by vibenedict — CHANGES_REQUESTED** [review]
LGTM 

Please fix conflict so I can merge.
I think you just need to revert changes made to this file in order to fix the conflicts

File: onchain/manifests/dev/base/contracts/starkludo-GameActions-72b46dc4.toml
Comment (vibenedict) [inline onchain/manifests/dev/base/contracts/starkludo-GameActions-72b46dc4.toml:1]: please revert change made to this file

File: onchain/manifests/dev/base/contracts/starkludo-PlayerActions-20dad3ac.toml
Comment (vibenedict) [inline onchain/manifests/dev/base/contracts/starkludo-PlayerActions-20dad3ac.toml:1]: please revert change made to this file

File: onchain/manifests/dev/base/contracts/starkludo-actions-6e36eb0d.toml
Comment (vibenedict) [inline onchain/manifests/dev/base/contracts/starkludo-actions-6e36eb0d.toml:1]: please revert change made to this file

**Review by vibenedict — CHANGES_REQUESTED** [review]

**Review by vibenedict — APPROVED** [review]
LGTM

**Pull request merged**